Networking/Networking/Model/Product/ProductCategory.swift

```diff
@@ -3,7 +3,7 @@ import Foundation
 
 /// Represents a ProductCategory entity.
 ///
-public struct ProductCategory: Decodable {
+public struct ProductCategory: Decodable, GeneratedCopiable {
```

Comment:
nit: is the `ProductCategory`'s `copy` function used in this PR?

Reply:
I'll use it in the next PR in my plans, but since it's not used here I'm removing it for the moment